src/main/java/org/spongepowered/common/mixin/core/entity/player/EntityPlayerMixin.java

```diff
@@ -361,6 +363,14 @@ public void onDeath(final DamageSource cause) {
         this.setFlag(0, false);
     }
 
+    @Redirect(method = "onLivingUpdate", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/World;getEntitiesWithinAABBExcludingEntity(Lnet/minecraft/entity/Entity;Lnet/minecraft/util/math/AxisAlignedBB;)Ljava/util/List;"))
+    private List<Entity> onSpongeGetEntitiesWithinAABBExcludingEntity(final World world, Entity entityIn, AxisAlignedBB bb) {
+        if (this.bridge$isUncollideable()) {
```

Comment:
[Should this depend on this key](https://github.com/SpongePowered/SpongeAPI/blob/stable-7/src/main/java/org/spongepowered/api/data/key/Keys.java#L2248-L2257)?

Reply:
`VANISH_IGNORES_COLLISION` is calling `bridge$isUncollideable` and `bridge$setUncollideable`

https://github.com/SpongePowered/SpongeCommon/blob/9be33cf2c38e66fe8ac774503ef88f0b33d51faa/src/main/java/org/spongepowered/common/data/processor/multi/entity/InvisibilityDataProcessor.java#L66

https://github.com/SpongePowered/SpongeCommon/blob/9be33cf2c38e66fe8ac774503ef88f0b33d51faa/src/main/java/org/spongepowered/common/data/processor/multi/entity/InvisibilityDataProcessor.java#L84